examples/lock-app/nrfconnect/main/include/AppTask.h

```diff
@@ -44,6 +44,7 @@ class AppTask
     void CancelTimer(void);
 
     void DispatchEvent(AppEvent * event);
+    void UpdateClusterState(void);
```

Comment:
void is unnecessary in the parameter list. Michael Spang pushed a commit recently that remove it from the whole codebase.